models/music_cont_in_proxy.cpp

```diff
@@ -167,11 +168,11 @@ nest::music_cont_in_proxy::get_status( DictionaryDatum& d ) const
 void
 nest::music_cont_in_proxy::set_status( const DictionaryDatum& d )
 {
-  Parameters_ ptmp = P_; // temporary copy in case of errors
-  ptmp.set( d, S_ );     // throws if BadProperty
+  Parameters_ ptmp = P_;    // temporary copy in case of errors
```

Comment:
Why do we need a temp attribute here? If something went wrong in the `set` function, then an exception will be thrown, and we end the execution.

Reply:
But then we might already have set half of the new parameters on P_, and the user has now way to know which worked and which did not.

This same pattern is used in every model:
1. create a temporary copy
1. check if setting on the copy works
1. if yes, swap the updated copy with the original parameters.

Reply:
@med-ayssar: can you please resolve this conversation if you are happy with my answer? We can only merge once all conversations are resolved.